### DISCUSSION TITLE
Add adventure setTitle (and originalTitle?) methods to inventory view

### DISCUSSION THREAD

**Discussion by kennytv** [PaperMC/Paper#10575] [Ideas]
Spigot's implementation isn't... *nice*, so we'll have to see if we just carbon copy it with components or change it up

**Comment by Machine-Maker** [comment]
I'm still firmly against this, and for deprecating the existing setTitle methods. The implementation is super scuffed and leads to inconsistencies when cancelling or not cancelling and dragging. @lynxplay tested this more in-depth to find those issues.
